sdk/test/Services/AutoScaling/IntegrationTests/Config/462/App.config

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="aws" type="Amazon.AWSSection, AWSSDK.Core" />
+  </configSections>
+  <appSettings>
+    <add key="AWSRegion" value="us-east-1" />
+    <add key="AWSProfileName" value="default" />
+    <add key="AWSLogging" value="SystemDiagnostics" />
+    <add key="AWSResponseLogging" value="OnError" />
+    <add key="AWSLogMetrics" value="true" />
+  </appSettings>
+  <system.diagnostics>
+    <trace autoflush="true" />
+    <sources>
+      <source name="Amazon">
+        <listeners>
+          <!--<add name="text" type="System.Diagnostics.TextWriterTraceListener" initializeData="tests-log.txt" />-->
+          <!--<add name="console" type="System.Diagnostics.ConsoleTraceListener" traceOutputOptions="DateTime"/>-->
+          <add name="console" type="AWSSDK_DotNet.IntegrationTests.Utils.ConsoleTraceListener, AWSSDK.IntegrationTestUtilities.NetFramework"/>
+        </listeners>
+      </source>
+    </sources>
+  </system.diagnostics>
+  <aws correctForClockSkew="true">
+    <s3 useSignatureVersion4="true"/>
+    <dynamoDB>
+      <dynamoDBContext>
```

Comment:
But speaking of configs, all of them include this `dynamoDBContext`... Should this only be included in the DynamoDB projects?

Reply:
I assume this is a leftover artifact of when the integ tests were all in a single project and we used tooling to split up the tests. Would make sense to only be in DynamoDB test projects but doesn't hurt.

Reply:
Got it. I ended up removing the `dynamoDB` section for all other services (https://github.com/aws/aws-sdk-net/pull/3202/commits/7ecbfa0fc6a7d97fa430a6c633e096888c9cf9c7) to avoid confusion later.